### PR TITLE
minor changes in exec_binary test case for Arm64

### PR DESCRIPTION
**Pull request by lubinszARM**
Signed-off-by: Bin Lu <bin.lu@arm.com>

### CODE REVIEW

**Comment by lubinszARM** [comment]
related issue: #2785

**Comment by lubinszARM** [comment]
Hi @avagin 
Any comments?
Thanks.

File: test/syscalls/linux/exec_binary.cc [hunk 1]
```diff
@@ -438,7 +438,11 @@ TEST(ElfTest, MissingText) {
   ASSERT_THAT(RetryEINTR(waitpid)(child, &status, 0),
               SyscallSucceedsWithValue(child));
   // It runs off the end of the zeroes filling the end of the page.
+#if defined(__x86_64__)
   EXPECT_TRUE(WIFSIGNALED(status) && WTERMSIG(status) == SIGSEGV) << status;
+#elif defined(__aarch64__)
+  EXPECT_TRUE(WIFSIGNALED(status) && WTERMSIG(status) == SIGILL) << status;
```
Comment (avagin) [inline test/syscalls/linux/exec_binary.cc:444]: could you add a comment why we get SIGILL here? Is 0 an invalid instruction opcode on arm64?

Comment (lubinszARM) [reply test/syscalls/linux/exec_binary.cc:444]: Yes. 0 is an invalid opcode.
Also, I have confirmed it in local with native mode.

Comment (lubinszARM) [reply test/syscalls/linux/exec_binary.cc:445]: Done. Comment was added.
Thanks.

**Pull request merged**